assets/javascripts/src/modules/analytics/setup.js

```diff
@@ -31,6 +31,7 @@ define([
     function setupAnalytics() {
         ophan.init();
         ga.init();
+        ophan.loaded.then(ga.init,ga.init);
```

Comment:
I'm just testing this, fwict it's firing both. Also you need to remove the ga.init() above!